books/RayTracingTheNextWeek.html

```diff
@@ -155,6 +144,15 @@
             );
         }
 
+      public:
+        double vfov       = 40;
+        double aperture   = 0;
+        double focus_dist = 10;
+
+        point3 lookfrom = point3(0,0,-1);
+        point3 lookat   = point3(0,0,0);
+        vec3   vup      = vec3(0,1,0);
+
```

Comment:
needs `time_start` and `time_end`. delete `time0` and `time1` below.

Reply:
Ooh — good catch!

Reply:
Might need to have a `camera.h` for each project in the future. Pain either way.